### ISSUE TITLE
Truncated infos on the Next page

### ISSUE THREAD

**Issue by oldman63** [Podcast-Republic/PodcastRepublic-iOS#32]
Date and xx% status is truncated on "Up Next" page

the same data are displayed perfectly on the "History" page

Next:
![IMG_0136](https://github.com/Podcast-Republic/PodcastRepublic-iOS/assets/3171626/34c1804f-ca72-4478-95b9-f40b0399d102)

History:
![IMG_0144](https://github.com/Podcast-Republic/PodcastRepublic-iOS/assets/3171626/f163d08c-d0fb-4a71-98d5-e63f1810802c)

**Comment by gemiren** [comment]
The history list uses two lines while the Up next list uses a single line so that the long text doesn't fit.

**Comment by oldman63** [comment]
Yes, this is clearly visible. If it stays that way, it's unfortunate.

**Comment by gemiren** [comment]
> Yes, this is clearly visible. If it stays that way, it's unfortunate.

Will think about that. Thanks!
